cpp/src/io/json/json_column.cu

```diff
@@ -601,6 +635,11 @@ table_with_metadata device_parse_nested_json(device_span<SymbolT const> d_input,
           return (user_dtypes.find(col_name) != std::end(user_dtypes))
                    ? user_dtypes.find(col_name)->second
                    : std::optional<schema_element>{};
+        },
+        [col_name](schema_element const& user_dtypes) -> std::optional<schema_element> {
+          return (user_dtypes.child_types.find(col_name) != std::end(user_dtypes.child_types))
+                   ? user_dtypes.child_types.find(col_name)->second
```

Comment:
`find(col_name)` should not be called twice.

Reply:
Great suggestion!